hi-ml-histopathology/src/histopathology/scripts/aggregate_metrics_crossvalidation.py

```diff
@@ -3,38 +3,101 @@
 #  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 #  ------------------------------------------------------------------------------------------
 
-"""
-Script to find mean and standard deviation of desired metrics from cross validation child runs.
-"""
-import os
+import sys
+from argparse import ArgumentParser
+from pathlib import Path
+from typing import List
+
 import pandas as pd
+from azureml.core import Run
+from azureml.core.run import _OfflineRun
 
-from health_azure import aggregate_hyperdrive_metrics, get_workspace
+HIML_ROOT = Path.cwd()
```

Comment:
This is very error prone. Can we assume for simplicity here that hi-ml has been installed locally in editable mode?

Reply:
Agreed this is messy but starting from file is problematic  in VS code since we have separated out the workspaces. Will send you a message offline to explain

Reply:
Agreed we need another solution to handle this. I couldn't get it to work with `Path(__file__).parent....` in VS Code since it  can't access any folder higher than `${workspaceFolder}` (which is now `hi-ml-histopathology`). Have sent you a message about this

Reply:
This is updated now